tests/tasks/test_exceptions.py

```
@@ -0,0 +1,333 @@
+"""
+Celeryタスク例外クラスのテスト
+"""
+import pytest
+from unittest.mock import Mock
+from tasks.exceptions import (
+    LLMError, TemporaryLLMError, PermanentLLMError,
+    RateLimitError, NetworkError, ServiceUnavailableError,
+    AuthenticationError, QuotaExceededError, InvalidRequestError,
```

Ruff (0.12.2)

`tasks.exceptions.QuotaExceededError` imported but unused

Remove unused import

(F401)